### PR TITLE
Docs: clarify the wording for language codes and adapt warnings

### PR DESCRIPTION
**Pull request by brandelune**
[SF#407 Clarify the status of language codes in project properties](https://sourceforge.net/p/omegat/documentation/407/)

### CODE REVIEW

File: doc_src/en/Introduction.xml [hunk 1]
```diff
@@ -250,20 +250,24 @@
...
 
-	  <para>See the <link linkend="introduction.install.spellchecker.dictionary"
-	  endterm="introduction.install.spellchecker.dictionary.title"/> section
-	  below for details.</para>
+	<warning>
+	  <para>Make sure that you enter the correct languages and that all other
+	  locations requiring a language code match the code you assign here. See
+	  the <link linkend="dialogs.project.properties.languages"
+	  endterm="dialogs.project.properties.languages.title"/> section for
+	  details.</para>
```
Comment (Kazephil) [inline doc_src/en/Introduction.xml:268]: Kept short to ensure there actually are further details in the linked section.

**Comment by Kazephil** [comment]
@brandelune 
I edited the files with proposed rephrasings and some comments.
Please have a look.

**Comment by brandelune** [comment]
I've rebased-squashed your modifs.

**Comment by Kazephil** [comment]
Merged into master.

**Pull request closed without merging**